nicknist/frontend-colors
# Hooks and Testing

What's this PR do?
This PR adds the hooks and testing for the components that used to be class components. In addition, there have been some updates to apiCalls and added functionality connected to the backend.

Where should the reviewer start?
The reviewer should pull this down, run npm test to make sure that those are rocking, and then run npm start in order to check the new features are working. You should be able to make projects, palettes, and only make projects when there are 3 palettes ready to go. Also run npm test to make sure that those are all passing.

How should this be manually tested?
Run the app and make sure that it's working the way that's intended. Also, run npm test to make sure that that's rolling.
